packages/compartment-mapper/src/import-hook.js

```diff
@@ -156,6 +158,21 @@ export const makeImportHookMaker = (
           // Archived compartments are not executed.
           return freeze({ imports: [], exports: [], execute() {} });
         }
+        if (exitModuleImportHook) {
+          console.error('#################i');
```

Comment:
Here again.

Reply:
yes, it's still early in the implementation and these are useful for showing when and what happens. I'll need them for the meeting.